src/gui/SubWindow.cpp

```diff
@@ -240,6 +241,27 @@ void SubWindow::setBorderColor( const QColor &c )
 
 
 
+
+int SubWindow::titleBarHeight() const
+{
+	QStyleOptionTitleBar so;
+	so.titleBarState = Qt::WindowActive; // kThemeStateActiv
+	so.titleBarFlags = Qt::Window;
+	return style()->pixelMetric(QStyle::PM_TitleBarHeight, &so, this);
```

Comment:
I was not exactly sure whether to specify the 2nd and 3rd arguments (for both `pixelMetric` calls), though it just works.